fdbserver/fdbserver.actor.cpp

```diff
@@ -646,6 +649,7 @@ static void printUsage(const char* name, bool devhelp) {
 		    "--kvfile FILE",
 		    "Input file (SQLite database file) for use by the 'kvfilegeneratesums' and 'kvfileintegritycheck' roles.");
 		printOptionUsage("-b [on,off], --buggify [on,off]", " Sets Buggify system state, defaults to `off'.");
+		printOptionUsage("-f [on,off], --fault_injection [on,off]", " Sets fault injection, defaults to `on'.");
```

Comment:
`-fi`? `-f` is used for test file already and please make sure if `-fi` works; otherwise, just use the long option flag.

Reply:
Just curious how it is different from `BUGGIFY` system?

Reply:
With `BUGGIFY` off, you can still have simulated process/machine failures that can cause recovery. This option is intended to ensure a failure-free execution that has no recovery.

Reply:
yes ```-fi``` works, I tested it. @jzhou77

Reply:
Sounds good. Please update usage with `-fi` option, instead of `-f`.

Reply:
Done.